### ISSUE TITLE
ORDO 4.0 syncronisation

### ISSUE THREAD

**Issue by matentzn** [monarch-initiative/mondo#4579]
This will probably be a bit more major, I can see ~300 new diseases. 

- I think we should try to get these in as a while you are still waiting on the ICD10 mapping stuff.
- Remember to be a bit cautious not to accidentally introduce duplicate terms that were somehow matched wrongly by the (old) Mondo slurping system.

[ordo_slurp.txt](https://github.com/monarch-initiative/mondo/files/8190536/ordo_slurp.txt)

**Comment by nicolevasilevsky** [comment]
action item:
- [ ] identify the terms in the ORDO list that should be obsolete and add the out of scope annotation and a comment about why it is out of scope and add them to Mondo as born obsolete. 
- [ ] Make a Mondo internal curation rule, ie phenotype, etc and add a seeAlso that links to a documentation page with our curation rules.

**Comment by nicolevasilevsky** [comment]
done! 🎉
